open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,34 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination by signaling to the service that they would
+        prefer requests to be paginated based on the number of items specified by pageSize.
```

Comment:
"...that they would prefer **response** to be paginated. If page-size is also specified, the service should return exactly the number of items specified by the page size."